appendix-basic65.tex

```diff
@@ -6723,10 +6719,10 @@ \subsection{RCURSOR}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$FE \$42
 \item [Format:] {\bf RCURSOR [colvar],[rowvar]}
-\item [Usage:]  Returns the current cursor column and cursor row.
+\item [Usage:]  Returns the current cursor column and row.
 
-\item [Remarks:] The row and column counts start at zero,
-                 the left most column is 0 and the top row is 0.
+\item [Remarks:] The row and column start at zero, where
```

Comment:
Perhaps replace with "Row and column values start at zero, ..."

Reply:
done